pom.xml

```diff
@@ -118,6 +118,14 @@
         </dependency>
 
         <dependency>
+            <!-- For logging on Windows -->
+            <groupId>org.fusesource.jansi</groupId>
+            <artifactId>jansi</artifactId>
+            <version>1.13</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Sorry for the delay (i was out of town). 

I don't think the issue is the lack of Erlang on Appveyor. They claim it's actually installed: (see https://www.appveyor.com/docs/installed-software/#erlang) and the integration test corroborates it since without it, RabbitMQ couldn't be started (see https://ci.appveyor.com/project/AlejandroRivera/embedded-rabbitmq/build/1.0.78-master)